src/libraries/System.IO.FileSystem/tests/File/GetSetAttributes_SafeFileHandle.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Tests
+{
+    public sealed class GetSetAttributes_SafeFileHandle : File_GetSetAttributes
```

Comment:
Since you are just extending the existing tests for `string path`, I suggest you just extend the same class in here.
```suggestion
    public sealed class GetSetAttributes_SafeFileHandle : FileGetSetAttributes
```